helpers/auth/authentication.py

```diff
@@ -91,7 +93,7 @@ def save_user(self):
             pass
         return True
 
-    def user_roles(self, *expected_args):
+    def user_roles(self, *expected_args):  # noqa: C901
```

Comment:
why just use `*args` instead of `*expected_args` then.

Reply:
@Bonifase There's a method within this method which is using *args